ext/web/02_event.js

```diff
@@ -972,7 +974,7 @@ class EventTarget {
 
     const { listeners } = self[eventTargetData];
 
-    if (!(ReflectHas(listeners, type))) {
+    if (!listeners[type]) {
```

Comment:
same here, use `in`

Reply:
`self[eventTargetData]` is only accessible internally so we can probably use whatever benchmarks fastest here. There shouldn't be a risk of improper boolean coercion if we know the field is `null | array`.

Reply:
`listeners` is an internal value which is only set by us, and it's either undefined or `[]`. There's no change in using object lookup or `in`, and since we have a lint rule against `in` I used object lookup.